src/main/java/com/github/reinert/jjschema/JsonSchemaGenerator.java

```diff
@@ -352,15 +352,15 @@ private void processPropertyCollection(Method method, Field field, ObjectNode sc
             } else {
                 genericClass = returnType;
             }
-            fowardReference = new ManagedReference(type, refAnn.value(), genericClass);
+            forwardReference = new ManagedReference(type, refAnn.value(), genericClass);
 
-            if (!isFowardReferencePiled(fowardReference)) {
-                pushFowardReference(fowardReference);
+            if (!isForwardReferencePiled(forwardReference)) {
+                pushForwardReference(forwardReference);
             } else
-//        	if (isBackwardReferencePiled(fowardReference)) 
+//        	if (isBackwardReferencePiled(forwardReference))
             {
-                boolean a = pullFowardReference(fowardReference);
-                boolean b = pullBackwardReference(fowardReference);
+                boolean a = pullForwardReference(forwardReference);
+                boolean b = pullBackwardReference(forwardReference);
```

Comment:
it looks like this change didn't take (somehow) -- `foward` lives on at https://github.com/reinert/JJSchema/blob/master/src/main/java/com/github/reinert/jjschema/JsonSchemaGenerator.java#L364

Reply:
ah looks like it conflicted with https://github.com/reinert/JJSchema/pull/79/files and 79 won :)